deploy/eck-operator/values.yaml

```diff
@@ -182,8 +182,8 @@ config:
     port: "0"
     # secureMode contains the options for enabling and configuring RBAC and TLS/HTTPs for the metrics endpoint.
     secureMode:
-      # secureMode.enabled specifies whether to enable RBAC and TLS/HTTPs for the metrics endpoint. (Will be enabled by default in v2.14.0)
-      # * This option requires using a ServiceMonitor to scrape the metrics and as such is mutually exclusive with the podMonitor.enabled option.
+      # secureMode.enabled specifies whether to enable RBAC and TLS/HTTPs for the metrics endpoint.
+      # * This option makes most sense when using a ServiceMonitor to scrape the metrics and is therefore mutually exclusive with the podMonitor.enabled option.
```

Comment:
I don't understand why we had an exclusion for pod monitors here? You can define TLS options on pod monitors in the same way as with service monitors. I don't see a reason why the scraping would not work with a pod monitor. @naemono

Reply:
https://github.com/elastic/cloud-on-k8s/pull/7567
https://github.com/prometheus-operator/prometheus-operator/issues/4523#issuecomment-1172107348

`PodMonitor`s do not have the `bearerToken` option, which is what I noted in the original PR where this was implemented. I will try it again and report back; maybe I was incorrect, but I do remember spending some time investigating this...

Reply:
That's a good point I did not check that.

Reply:
```
W1210 10:17:39.199761 2794893 warnings.go:70] unknown field "spec.podMetricsEndpoints[0].bearerTokenFile"
```
Verified: This will not work with `podMonitor`.